apps/web/app/(authenticated)/(app)/app/apis/client.tsx

```diff
@@ -26,6 +27,9 @@ export function ApiList({ apis }: { apis: ApiWithKeys }) {
   const [localData, setLocalData] = useState(apis);
   return (
     <div>
+      <Suspense>
+        <PostHogAuthIdentify />
```

Comment:
how does this interact with clerk? oO

could we add it way higher in the layouts somewhere?

Reply:
We want it here, so it runs only on the client, it's unfortunately the first client side page that has auth state attached. 

Not sure what you mean by how does it interact with Clerk? Clerk has Auth state anywhere inside /app/(app)/authenticated so you can access it wherever.